play/src/main/scala/com/managementapp/services/common/ProtobufUtil.scala

```diff
@@ -13,4 +13,9 @@ trait ProtobufUtil {
     JsonFormat.parser().merge(json, builder)
     builder.build.asInstanceOf[T]
   }
+
+  def generateJSON[M <: Message](content: M): String = {
```

Comment:
The func name is a bit confusing the first time I saw it. Maybe pbToJson() is clearer? And the same for the one above. (jsonToPb() to keep them in consistent)